server/src-lib/Hasura/Server/Logging.hs

```diff
@@ -192,7 +191,10 @@ data OperationLog
   { olRequestId          :: !RequestId
   , olUserVars           :: !(Maybe UserVars)
   , olResponseSize       :: !(Maybe Int64)
+  , olQueryIoTime        :: !(Maybe Double)
```

Comment:
What is `query_io_time`? From what I followed, is it the time taken for the request body to be read?

Reply:
Yep, can the comment be improved?

I'm towards the end of a big refactor and rebase though, so some details might change. A question: do you think it's important to also include the IO wait time for the send of the response to the client? For whatever reason I didn't think to do this, but since there are often different bandwidth limits on uploads and downloads it might be a good idea. 

What do you think?

Reply:
I couldn't see an obvious way to time sending the response to the client so haven't done that below. I can dig further into spock if that seems important.